packages/graph/.env

```diff
@@ -0,0 +1,17 @@
+HASH_GRAPH_PG_USER=postgres
+HASH_GRAPH_PG_PASSWORD=postgres
+HASH_GRAPH_PG_HOST=localhost
+HASH_GRAPH_PG_PORT=5432
+HASH_GRAPH_PG_DATABASE=graph
```

Comment:
I don't mind this but why do we keep changing the name of the DB? From `postgres` to `postgres_graph` to `graph` now...

Reply:
We are already in a `postgres` database, I see no reason to use this in the database name, too.